src/protected/ProtectedAccountMessage/__tests__/ProtectedAccountMessage.test.tsx

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs';
+
+const ADMINUI_START_PAGE_PATH =
+  'src/pages/[platform]/tools/console/adminui/start/index.mdx';
+
+describe('Protected Account Message', () => {
+  it('should render ProtectedAccountMessage component on the Admin UI Start page', async () => {
```

Comment:
Can you please add the same comment as previous PR's about not removing this test?